src/constants/community.ts

```diff
@@ -0,0 +1,7 @@
+export const GRID_ITEM_PER_PAGE = 40;
+export const LIST_ITEM_PER_PAGE = 200;
+// WIP: Temporary for testing
+export const ART_GOBBLER_ADDRESS = [
+  '0xc729ce9bf1030fbb639849a96fa8bbd013680b64',
+  '0x5180db8f5c931aae63c74266b211f580155ecac8',
+];
```

Comment:
could we name this `GRID_ENABLED_COMMUNITY_ADDRESSES`

and also comment above each contract address with the name

e.g.
```
// crypto coven
'0x5180db8f5c931aae63c74266b211f580155ecac8',
```

Reply:
Updated!